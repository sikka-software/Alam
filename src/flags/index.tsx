export * from './SA';
export * from './USA';
