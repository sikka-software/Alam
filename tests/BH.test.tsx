import React from 'react';
import renderer from 'react-test-renderer';
import { expect, it } from 'vitest';
import { BH } from '../src';

it('renders correctly', () => {
  const tree = renderer.create(<BH />).toJSON();
  expect(tree).toMatchSnapshot();
});
