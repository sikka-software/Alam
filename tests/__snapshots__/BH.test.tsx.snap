// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders correctly 1`] = `
<div>
  <svg
    height="15px"
    version="1.1"
    viewBox="0 0 21 15"
    width="21px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <title>
      BH
    </title>
    <desc>
      Flag of Bahrain
    </desc>
    <defs>
      <linearGradient
        id="linearGradient-1"
        x1="50%"
        x2="50%"
        y1="0%"
        y2="100%"
      >
        <stop
          offset="0%"
          stopColor="#FFFFFF"
        />
        <stop
          offset="100%"
          stopColor="#F0F0F0"
        />
      </linearGradient>
      <linearGradient
        id="linearGradient-2"
        x1="50%"
        x2="50%"
        y1="0%"
        y2="100%"
      >
        <stop
          offset="0%"
          stopColor="#E7243B"
        />
        <stop
          offset="100%"
          stopColor="#CC162C"
        />
      </linearGradient>
    </defs>
    <g
      fill="none"
      fillRule="evenodd"
      id="Symbols"
      stroke="none"
      strokeWidth="1"
    >
      <g
        id="BH"
      >
        <rect
          fill="url(#linearGradient-1)"
          height="15"
          id="FlagBackground"
          width="21"
          x="0"
          y="0"
        />
        <rect
          fill="url(#linearGradient-2)"
          height="15"
          id="Mask-Copy"
          width="21"
          x="0"
          y="0"
        />
        <polygon
          fill="url(#linearGradient-1)"
          id="Rectangle-2-Copy"
          points="0 15 7.5 15 6 14 7.5 13 6 12 7.5 11 6 10 7.5 9 6 8 7.5 7 6 6 7.5 5 6 4 7.5 3 6 2 7.5 1 6 0 0 0"
        />
      </g>
    </g>
  </svg>
</div>
`;
